algopy_testing/tests/artifacts/PrimitiveOps/contract.py

```diff
@@ -0,0 +1,55 @@
+from algopy import ARC4Contract, Bytes, UInt64, arc4, op
+
+
+class PrimitiveOpsContract(ARC4Contract):
+    @arc4.abimethod()
+    def verify_uint64_init(self, raw_value: Bytes) -> UInt64:
+        return op.btoi(raw_value)
+
+    @arc4.abimethod()
+    def verify_uint64_add(self, a: UInt64, b: UInt64) -> UInt64:
+        return a + b
```

Comment:
if we split these into 
```python
result = a <op> b
return result
```
We can verify the failures occur at the right spot, not just at the end of the function

Reply:
Have made this change, will leave the work of verifying failures occur at the correct position for later (once subroutines and/or contracts are testable)